extruct/jsonld.py

```diff
@@ -21,10 +22,12 @@ def extract(self, htmlstring, url='http://www.example.com/', encoding="UTF-8"):
         return self.extract_items(lxmldoc)
 
     def extract_items(self, document, *args, **kwargs):
-        return [item for items in map(self._extract_items,
-                                      self._xp_jsonld(document))
-                     for item in items
-                         if item]
+        try:
+            return [item for items in map(self._extract_items,
+                                          self._xp_jsonld(document))
```

Comment:
There could be only one JSONLD that is empty in the page, and this would swallow the potentially succesful extruction.
`_extract_items` is probably a better place for this catch.